tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "inlineSourceMap": true,
     "inlineSources": true,
     "module": "ESNext",
-    "target": "ES6",
+    "target": "ES2018",

```

Comment:
This will cause the plugin to stop working on older Obsidian installations.
Please keep ES6 compatibility.